packages/webapp/src/elections/components/ongoing-election-components/round-header.tsx

```diff
@@ -1,73 +1,35 @@
 import { useState } from "react";
-import dayjs from "dayjs";
+import dayjs, { Dayjs } from "dayjs";
 import { FaCheckCircle } from "react-icons/fa";
 import { GoSync } from "react-icons/go";
 
-import { Text } from "_app/ui";
-import { VotePieMer } from "elections";
-
-type RoundHeaderProps =
-    | {
-          roundEndTime: string;
-          roundIndex: number;
-          subText?: string;
-          headlineText?: string;
-      }
-    | {
-          roundEndTime?: string;
-          roundIndex: number;
-          subText: string;
-          headlineText?: string;
-      };
+type RoundHeaderProps = {
+    isRoundActive: boolean;
+    headlineComponent: React.ReactNode;
+    sublineComponent: React.ReactNode;
+    children?: React.ReactNode;
+};
 
 export const RoundHeader = ({
```

Comment:
This poor component was just trying to do _too_ much. Made him more presentational.

Reply:
+1!